flaskbb/forum/views.py

```diff
@@ -1108,6 +1110,89 @@ def post(self, mode=None):
         preview = renderer(text)
         return preview
 
+def user_get_file_max_length():
+    user_groups_file_max_length=[current_user.primary_group.uploaded_file_max_length]
+    for group in current_user.secondary_groups:
+        user_groups_file_max_length.append(group.uploaded_file_max_length, )
+    user_groups_file_max_length.append(current_app.config["MAX_UPLOAD_SIZE"])
+
+    if -1 in user_groups_file_max_length:
+        return -1
+    else:
+        return max(user_groups_file_max_length)
+
+def user_get_upload_folder_limit():
+    user_groups_upload_folder_limit=[current_user.primary_group.upload_folder_limit]
```

Comment:
```suggestion
    user_groups_upload_folder_limit = current_user.primary_group.upload_folder_limit
```
Сразу видно человека, что не решал 27 задание ОГЭ по информатики...